device/demo/speech_locales.gd

```diff
@@ -0,0 +1,6 @@
+# Define your game's speech/voice locales within the game
+# because ProjectSettings doesn't deal well with arrays.
+# These can be used in your game to change the spoken language.
```

Comment:
Indeed. I believe Godot supports `PoolStringArray`s internally, but they don't seem to be editable from the project settings menu, so this seems like a good solution.